packages/orchestrator/internal/consts.go

```diff
@@ -0,0 +1,15 @@
+package internal
+
+import (
+	"github.com/e2b-dev/infra/packages/shared/pkg/env"
+)
+
+const (
+	// Private IP don't leave the sandbox through the network bridge, so we use a reserved IP address for it:
+	// See TEST-NET-3 on https://en.wikipedia.org/wiki/Reserved_IP_addresses
+	defaultSandboxEventIP = "203.0.113.0"
```

Comment:
Can we use IP from local link range? Same range used by metadata services in Firecracker/EC2. I think they are fitting better what way are trying to do here.

https://en.m.wikipedia.org/wiki/Link-local_address